test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules.lua

```diff
@@ -0,0 +1,439 @@
+---------------------------------------------------------------------------------------------------
+-- RC common module for AUDIO, LIGHT, HMI_SETTINGS modules
+---------------------------------------------------------------------------------------------------
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+
+--[[ Required Shared libraries ]]
+local commonRC = require("test_scripts/RC/commonRC")
+local test = require("user_modules/dummy_connecttest")
+
+--[[ Local Variables ]]
+local c = {}
+c.modules = { "RADIO", "CLIMATE", "AUDIO", "LIGHT", "HMI_SETTINGS" }
```

Comment:
@GetmanetsIrina Suggest to reduce modules list by the ones related to proposal.
We have already have test scripts for RADIO and CLIMATE modules and they a planned to be run in scope of regression.

Reply:
@GetmanetsIrina Please ignore previous comment since proposal is related to the new parameters for existing RADIO and CLIMATE modules